src/dmd/typesem.d

```diff
@@ -205,7 +207,7 @@ private extern (C++) final class TypeSemanticVisitor : Visitor
         mtype.basetype = mtype.basetype.toBasetype().mutableOf();
         if (mtype.basetype.ty != Tsarray)
         {
-            mtype.error(loc, "T in __vector(T) must be a static array, not %s", mtype.basetype.toChars());
+            mtype.error(loc, "T in __vector(T) must be a static array, not `%s`", mtype.basetype.toChars());
```

Comment:
Shouldn't `__vector(T)` be surrounded by tick marks also?

Reply:
There are many such decisions, and they don't always have clear answers. In this case, `T` is not a user defined type, it never appears in his source code.

I opted instead to make these changes a bulk "first approximation", taking the most straightforward approach in order to get the bulk of it done.

Reply:
Ok, I understand.